app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala

```diff
@@ -48,14 +49,22 @@ class AuditService @Inject()(auditConnector: DefaultAuditConnector) {
     sendExplicitAuditEvent(CaseReleased, toJson(c))
   }
 
+  def auditCaseCompleted(c: Case)
+                       (implicit hc: HeaderCarrier): Future[Unit] = {
```

Comment:
`Unit`

Reply:
For consistency. All the connectors we use have a `Future` return type, this should be no different. Under the hood it is a thread blocking HTTP request so sticking it in a future means we can parallelise it.
Also improves the Service layer code having it as a future